### PR TITLE
Book library homepage using Javascript.

### PR DESCRIPTION
**Pull request by Abdoulaye-Thespy**
In this milestone, I create a Library book app with Vanilla JavaScript, Array mylibrary which contain books objects get stored on local storage which in an optional requirement for this project.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+  let author = document.getElementById('orangeForm-author').value
+  let numberOfPages = document.getElementById('orangeForm-np').value;
+  let newBook = new Book(name, author, numberOfPages);
+  myLibrary.push(newBook);
+  console.log(myLibrary);
+  render(name, author, numberOfPages);
+  
+  save();
+}
+
```
Comment (shloch) [inline index.js]: - control form values before you submit/push (security)
- remove `console.log`  [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/console_log_requirement.png)]

File: index.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    }
+  }
+}
+
+readBtn.onclick = function delete_book(e) {
+      if(e.target.className=='btn btn-success'){
+        e.target.innerHTML="READ";
+  }
+}
+
```
Comment (shloch) [inline index.js]: - you didn't also delete book from local storage

File: index.js [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+ for (let i=0; i < myLibrary.length; i++) {
+     let name = myLibrary[i].name;
+     let author = myLibrary[i].author;
+     let numberOfPages = myLibrary[i].numberOfPages;
+     let readstatus = myLibrary[i].readStatus;
+     console.log(readstatus);
```
Comment (shloch) [inline index.js]: remove `console.log`

File: index.js [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+function fetch() {
+if (localStorage.getItem('books') == null){
+     localStorage.setItem('books', JSON.stringify([]));
+     return JSON.parse(localStorage['books']);
+}
+else {
+ return JSON.parse(localStorage['books']);
+ console.log("inside fetch");
+}
+}
```
Comment (shloch) [inline index.js]: - this code is not well formatted, please check linters again (or format cleanly)
- remove `console.log`

**Review by shloch — CHANGES_REQUESTED** [review]
## Hello @Abdoulaye-Thespy 

Great job with your work
App works perfect to about 98%
Let's get to some debugging

### [changes required]
- "empty" form can be submitted, please control this
lib

- your `js` file is not well formatted, please work on making that code clean

- plenty `console.log` to remove...take care of that

**Review by uryelah — CHANGES_REQUESTED** [review]
Hi **Abdoulaye Razak**, I'm Sarah your reviewer for this PR 👩‍🚀 

Good job! Now just make sure your linter checks are all passing.

Check the pointers below and keep on coding! 🏃 💨 

# Required Changes 🏗️ 🐛 💡

- [ ] Make sure there are no linter errors being outputted by the Github Actions checker.

- [ ] I know you added the .gitignore with node_modules but make sure to also delete de node_modules directory from your repo. 

-----

 **Kindly fix the pointed out issues and comments below before requesting the next code review**

------------

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Review by bellom — APPROVED** [review]
## Hi @Abdoulaye-Thespy  :clap: 

**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit the project completion form.
Good luck in the next project :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**